### ISSUE TITLE
Need support to open gRPC service only on localhost

### ISSUE THREAD

**Issue by softnado** [dapr/python-sdk#593]
https://github.com/dapr/python-sdk/blob/5904cff8c50aeaf578b870b010b6b8e511765d23/ext/dapr-ext-grpc/dapr/ext/grpc/app.py#L69C8-L69C8

**Comment by berndverst** [comment]
@softnado I have no idea what this request is about. It is not actionable. Please update the title and description and provide details.

The Python SDK repo has 5 different SDKs and covers many different scenarios:

`dapr` - the Python SDK (includes workflow management)
`dapr-ext-grpc` - the gRPC server for pubsub and input binding events
`dapr-ext-fastapi` - the Fast API server for pubsub and input binding events
`flask-dapr` - the Flask API server for pubsub and input binding events
`dapr-ext-workflow` the Alpha SDK for **Workflow authoring**.

**Comment by berndverst** [comment]
I assume this is what you want to change:

https://github.com/dapr/python-sdk/blob/5904cff8c50aeaf578b870b010b6b8e511765d23/ext/dapr-ext-grpc/dapr/ext/grpc/app.py#L65-L71

**Comment by berndverst** [comment]
Sorry @softnado I didn't resolve your link originally. I see what you want now - but next time please do provide more details in the description please.

**Comment by berndverst** [comment]
@softnado as a temporary workaround, please do not call `app.run(PORT)`, but instead manually do:

```python
app_port = 5678
app._server.add_insecure_port(f'localhost:{app_port}')
self._server.start() 
self._server.wait_for_termination() 
```

This is the `grpc` library method `add_insecure_port` and whatever syntax they support is what we support here: https://grpc.github.io/grpc/python/grpc.html#grpc.Server.add_insecure_port

I might not have time to make this change in the upcoming release for the Python SDK (1.11) but we can definitely get it into the next version.

This is your simple solution for your situation in the meantime.

**Comment by berndverst** [comment]
This is now available in SDK 1.12.0 - there is a new optional paramater to specify the host IP
